app/helpers/session_timeout_warning_helper.rb

```diff
@@ -1,14 +1,14 @@
 module SessionTimeoutWarningHelper
   def session_timeout_frequency
-    (AppConfig.env.session_check_frequency || 150).to_i
```

Comment:
This and `session_timeout_start` had their intended defaults switched I think? `application.yml.default` has a frequency of 30 and a delay of 150, which is the inverse of what it has here. Since these configurations were set in all environments, it should be safe to remove the fallback values in the class, even though they're different.